src/app/events/org-events.component.ts

```diff
@@ -92,24 +97,32 @@ export class OrgEventsComponent implements OnInit {
           })),
       }
 
+    let tabs = []
+    if (this.isPublicView()) {
```

Comment:
I'll look into it in the next cycle after the feature is restricted for civil only :)